lyrics-core/src/main/java/com/flipkart/lyrics/annotators/GsonStyle.java

```diff
@@ -54,4 +54,9 @@ public void processSubTypeRule(TypeSpec.Builder typeSpec, TypeModel typeModel) {
 
     }
 
+    @Override
+    public void processPropertyOrderRule(TypeSpec.Builder typeSpec, TypeModel typeModel) {
+
```

Comment:
yes Gson does it differently (doesn't have annotations I believe), it can be added here later if required